tests/TestData/image.txt

```diff
@@ -0,0 +1,101 @@
+4.791809315615899284e-06 5.126409095240729533e-06 5.482344817439967818e-06 5.860436828263794873e-06 6.261427949991271167e-06 6.685958519346216250e-06 7.134538159388992342e-06 7.607514276478231539e-06 8.105037394309616189e-06 8.627023590304862372e-06 9.173114487700837635e-06 9.742635479962969059e-06 1.033455312047779636e-05 1.094743289431602767e-05 1.157939889040375566e-05 1.222809719686422884e-05 1.289066512923155352e-05 1.356370864470943822e-05 1.424329046461854813e-05 1.492493148699121162e-05 1.560362798609530102e-05 1.627388683200297291e-05 1.692978049454232795e-05 1.756502290753031944e-05 1.817306636396866240e-05 1.874721851590919604e-05 1.928077731418092940e-05 1.976718041899953290e-05 2.020016434112093881e-05 2.057392744872728462e-05 2.088329011647891556e-05 2.112384481001813223e-05 2.129208887763410244e-05 2.138553330892089228e-05 2.140278171794446455e-05 2.134357526257278859e-05 2.120880101858876689e-05 2.100046334266056049e-05 2.072161981318451144e-05 2.037628525872455594e-05 1.996930901251342780e-05 1.950623174463551830e-05 1.899312894446564563e-05 1.843644833083650007e-05 1.784284818363337784e-05 1.721904288853440447e-05 1.657166096739295431e-05 1.590711964690762802e-05 1.523151871551142664e-05 1.455055513944587314e-05 1.386945874043474197e-05 1.319294824116760867e-05 1.252520619707070214e-05 1.186987076593489334e-05 1.123004191355590767e-05 1.060829949176512369e-05 1.000673062408193473e-05 9.426963958421185049e-06 8.870208560282601460e-06 8.337295490736849356e-06 7.828720412929230757e-06 7.344685875822823757e-06 6.885142217221749505e-06 6.449826297764277464e-06 6.038297516197357142e-06 5.649970760271200394e-06 5.284146116368796390e-06 4.940035295966056272e-06 4.616784841034183666e-06 4.313496247184693751e-06 4.029243196693843538e-06 3.763086127520648060e-06 3.514084382819899351e-06 3.281306191671173177e-06 3.063836728785349924e-06 2.860784491348285082e-06 2.671286217018125263e-06 2.494510550090621035e-06 2.329660644313398244e-06 2.175975871768711024e-06 2.032732788400213717e-06 1.899245488655577039e-06 1.774865464704814865e-06 1.658981069988006503e-06 1.551016672554831752e-06 1.450431570814491929e-06 1.356718732893913727e-06 1.269403410743036948e-06 1.188041671340356859e-06 1.112218879738977483e-06 1.041548162144248555e-06 9.756688716189628494e-07 9.142450742644717397e-07 8.569640697244255721e-07 8.035349565075150114e-07 7.536872498407045467e-07 7.071695574662723474e-07 6.637483169152292940e-07 6.232065962640657177e-07 5.853429591570997427e-07 5.499703939055769961e-07
```

Comment:
this is in .txt file. Perhaps you can save it as a numpy file, such as np.save() np.load()?